ui/components/app/snaps/snap-ui-renderer/snap-ui-renderer.js

```diff
@@ -131,6 +131,10 @@ const SnapUIRendererComponent = ({
         className="snap-ui-renderer__content"
         height={BlockSize.Full}
         backgroundColor={contentBackgroundColor}
+        style={{
```

Comment:
With this, the margin around the Box in the Container is not scrollable, so we can't scroll to the bottom entirely.

![image](https://github.com/user-attachments/assets/889732ab-d72b-4154-bceb-66cae1c83f2b)